jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/RuntimeGroup.kt

```diff
@@ -17,108 +16,55 @@ import com.intellij.openapi.projectRoots.SdkType
 import com.intellij.openapi.roots.ModuleRootManager
 import com.intellij.openapi.roots.ProjectRootManager
 import com.intellij.openapi.util.KeyedExtensionCollector
-import com.intellij.openapi.util.LazyInstance
-import com.intellij.util.KeyedLazyInstance
-import com.intellij.util.xmlb.annotations.Attribute
 import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.jetbrains.core.IdBasedExtensionPoint
 
 /**
  * Grouping of Lambda [Runtime] by parent language.
  *
- * A Lambda [Runtime] belongs to a single [RuntimeGroup], a [RuntimeGroup] may have several
- * Lambda [Runtime]s, [Language]s or [Sdk]s.
+ * A Lambda [Runtime] belongs to a single [RuntimeGroup], a [RuntimeGroup] may have several Lambda [Runtime]s, [Language]s or [Sdk]s.
  */
-enum class RuntimeGroup {
-    JAVA,
-    PYTHON,
-    NODEJS,
-    DOTNET;
-
-    private val info by lazy {
-        RuntimeGroupInformation.getInstances(this)
-    }
+interface RuntimeGroup {
```

Comment:
To fix the equality issue, this will become an abstract class. I am going to model it after `Language` if curious